bin/configure.sh

```diff
@@ -27,3 +27,82 @@ export NODE_PATH=../..:../../zlux-server-framework/node_modules:$NODE_PATH
 __UNTAGGED_READ_MODE=V6 $NODE_BIN initInstance.js
 
 APP_WORKSPACE_DIR=${INSTANCE_DIR}/workspace/app-server
+
+# Add static definition for zss. TODO: Needs routedServices + apiInfo
+cat <<EOF >${STATIC_DEF_CONFIG_DIR}/zss.ebcidic.yml
+#
+services:
+  - serviceId: zss
+    title: Zowe System Services Server (ZSS)
```

Comment:
That would be ZSSS
You could just have it as
Zowe System Services (ZSS)

Reply:
oops